features/cellular/framework/AT/AT_CellularContext.cpp

```diff
@@ -614,6 +644,22 @@ void AT_CellularContext::ppp_status_cb(nsapi_event_t ev, intptr_t ptr)
     _device->cellular_callback(ev, ptr);
 }
 
+void AT_CellularContext::ppp_disconnected()
+{
+#if (DEVICE_SERIAL && DEVICE_INTERRUPTIN)
```

Comment:
I don't think this is conditional on that - there are other possible implementations of `enable_hup`. Could certainly work over USB serial.

Reply:
removed #ifdef